plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -739,22 +769,28 @@ private static function update() {
 					);
 					++$loop;
 				}
+
+				wc_get_logger()->info(
+					sprintf( '  Updates from version %s scheduled.', $version ),
```

Comment:
```suggestion
					sprintf( 'Updates from version %s scheduled.', $version ),
```
Are these extra spaces in the front intentional?

Reply:
Yes, so that subtasks look "indented" in the logs:

<img width="737" alt="Screenshot 2025-02-26 at 19 45 49" src="https://github.com/user-attachments/assets/52c71b4b-1166-42f8-a63f-4a6c1098e591" />

A similar strategy is used by WP's auto plugin updates, but happy to adjust if you think it's confusing.

Reply:
Makes sense. Good to have this. :unagi: xD